api/desecapi/models/mfa.py

```diff
@@ -56,7 +56,7 @@ def base32_secret(self):
     @property
     def uri(self):
         return self._totp.provisioning_uri(
-            name=self.name,
+            name=self.user.email,
```

Comment:
Won't that break backwards compatibility?

Reply:
With what? The URI ist only conveyed once (and displayed in the QR code) when setting up the TOTP.